devicesetup/src/main/java/io/particle/android/sdk/devicesetup/SetupCompleteIntentBuilder.java

```diff
@@ -0,0 +1,9 @@
+package io.particle.android.sdk.devicesetup;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.Nullable;
+
+public interface SetupCompleteIntentBuilder {
+    Intent buildIntent(Context ctx, @Nullable SetupResult result);
```

Comment:
I feel that that will make it too awkward to implement - I would hope that most people are aware of the problem with anonymous inner classes in activities and so would try to avoid this. 

Speaking of which, I should update the example to avoid this very issue...